service/worker/esanalyzer/domainWorkflowTypeCountWorkflow.go

```diff
@@ -146,51 +146,64 @@ func (w *Workflow) emitWorkflowTypeCountMetrics(ctx context.Context) error {
 			return err
 		}
 		for _, domainName := range workflowMetricDomainNames {
-			wfTypeCountEsQuery, err := w.getDomainWorkflowTypeCountQuery(domainName)
-			if err != nil {
-				logger.Error("Failed to get ElasticSearch query to find domain workflow type Info",
-					zap.Error(err),
-					zap.String("DomainName", domainName),
-				)
-				return err
+			switch w.analyzer.readMode {
+			case ES:
+				err = w.emitWorkflowTypeCountMetricsES(ctx, domainName, logger)
+			default:
+				err = w.emitWorkflowTypeCountMetricsES(ctx, domainName, logger)
```

Comment:
both cases doing to same thing. did you mean to emit a metric for pinot here

Reply:
Yes. I'll add pinot in the next PR